src/main/java/org/folio/persist/IterationJobRepository.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.persist;
+
+import static org.folio.rest.persist.PgUtil.postgresClient;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+import org.folio.rest.jaxrs.model.IterationJob;
+
+public class IterationJobRepository extends AbstractRepository<IterationJob> {
+
+  public static final String TABLE_NAME = "iteration_job";
```

Comment:
Why is this public? Is it used externally to the repository?

Reply:
Yes, it's used in `InstanceIterationAPI`

Reply:
Could that code be moved to within the repository, meaning this can be kept private?

Reply:
Updated. Thanks